### PR TITLE
Add notify-send

### PR DESCRIPTION
**Pull request by rakeshdas1**

### CODE REVIEW

File: pages/linux/notify-send.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Notify-send
+
+Uses the current desktop environment's notification system to create a notification. This should work on many modern desktop environments such as XFCE, GNOME, and Cinnamon.
```
Comment (igorshubovych) [inline pages/linux/notify-send.md]: IMO, sentence "This should work on many modern desktop environments such as XFCE, GNOME, and Cinnamon." should not be here. If users are looking for it, then they have this command.

File: pages/linux/notify-send.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Notify-send
```
Comment (igorshubovych) [inline pages/linux/notify-send.md]: Should be lowercase

**Comment by igorshubovych** [comment]
Plz squash commits.

**Comment by rubenvereecken** [comment]
Alright @rakeshdas1 could you please squash your commits? Too many of them just say the same.

**Comment by rubenvereecken** [comment]
Okay so apparently it's really not easy to pull in just a single commit and I still want to give you credit so let's try this once more @rakeshdas1, I'll be nice :). Just copy the page as it currently is and commit it onto a clean branch. Just start fresh from master, paste it there. I don't even care if it's squashed.

Since it's too hard to pull in just that commit I want we'll have to submit our own version of this :( And this command is really useful.

**Pull request closed without merging**